manifests/modules/aiml/chatbot/ray-service-llama2-chatbot/ray-service-llama2.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: llama2
+
+---
+apiVersion: ray.io/v1
+kind: RayService
+metadata:
+  name: llama2
+  namespace: llama2
+spec:
+  serviceUnhealthySecondThreshold: 900
+  deploymentUnhealthySecondThreshold: 300
+  serveConfigV2: |
+    applications:
+    - name: llama2
+      import_path: "ray_serve_llama2:entrypoint"
+      runtime_env:
+        env_vars:
+          MODEL_ID: "NousResearch/Llama-2-13b-chat-hf"
+          NEURON_CC_FLAGS: "-O1"
+          LD_LIBRARY_PATH: "/home/ray/anaconda3/lib:$LD_LIBRARY_PATH"
+          NEURON_CORES: "24"
+      deployments:
+        - name: Llama-2-13b-chat-hf
+          autoscaling_config:
+            metrics_interval_s: 0.2
+            min_replicas: 1
+            max_replicas: 1
+            look_back_period_s: 2
+            downscale_delay_s: 30
+            upscale_delay_s: 2
+            target_num_ongoing_requests_per_replica: 1
```

Comment:
add a comment that this field is deprecated and will need to be updated when DoEKS updates